woocommerce/woocommerce
# Fix a typo in the SKU of sample products

### Changes proposed in this Pull Request:

It bothered me several times when creating sample product data from the UI in WooCommerce that the sku is misspelled which looks unprofessional. This fixes that by changing `fasion` to `fashion`.

There is no reference to `fasion` anywhere else in the codebase in e2e tests or otherwise, so I can't foresee this causing any kind of problem.




### How to test the changes in this Pull Request:

I believe that seeing e2e tests and other CI checks pass should be sufficient. You could also double check and confirm like I have that no instance of `fasion` is referenced elsewhere in the codebase.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Fix a typo in the Woo product sample data

</details>
